setup.py

```diff
@@ -1,8 +1,13 @@
 from setuptools import setup, find_packages
 
 setup(
-    name = "regulations",
-    version = "0.1.0", 
-    license = "public domain", 
-    packages = find_packages()
+    name="regulations",
+    version="2.0.0", 
+    license="public domain", 
```

Comment:
You might want to add CC0 as well, as public domain isn't a thing outside the US.